src/content/docs/quick-start/building-for-skale.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Building for SKALE
+description: Learn how to build on SKALE
+---
+import { SelectChain } from "../../../components/SelectChain";
+import { Card, CardGrid, Steps, Tabs, TabItem } from "@astrojs/starlight/components";
+
+SKALE Chains and the greater SKALE Network offer a number of benefits for developers.
+The following page goes through a number of key topics about SKALE to help you build successfully.
+
+### EVM Compatability
+
+The SKALE Netowrk is a Layer-1 blockchain network that contains many individual SKALE Chains.
+Every SKALE Chain is fully EVM compatible meaning that all of your favorite tools on Etheruem,
+your existing smart contracts, and all of your scripts should work out of the box.
+
+### Zero Gas Fees
+
+SKALE is known for it's zero gas fee model, but developers are often unware of the strategic decisions
+made at the chain level to make that happen. Every SKALE Chain contains a native token called SKALE Fuel (sFUEL).
+**sFUEL** is almost identical to **Ether** on Ethereum with the largest difference being that sFUEL by default has
+no monetary value or available market.
+
+### How to Take Payments
+
+It is quite common for dApps to use various mechanisms to represent value on chains.
+It is common to see many dApps use the native gas token of a chain -- Ether for example --
+to allow for payment. With sFUEL holding no monetary value, it means that alternative forms of value must be used.
+The most common way is to use ERC-20 tokens to represnet value.
+
+> #### Developer Tip
```

Comment:
Shouldn't be used instead the same pattern we're using in other developer notes?

:::tip[Developer Note]
:::

Reply:
Resolved by [af389b8](https://github.com/skalenetwork/docs.skale.space/pull/108/commits/af389b83a86d089e6066f96cac399adcd982686c)